gokart/task.py

```diff
@@ -77,7 +77,10 @@ def requires(self):
         return tasks or []  # when tasks is empty dict, then this returns empty list.
 
     def make_task_instance_dictionary(self) -> Dict[str, 'TaskOnKart']:
-        return {key: var for key, var in vars(self).items() if isinstance(var, TaskOnKart)}
+        return {key: var for key, var in vars(self).items() if isinstance(var, TaskOnKart) or self._is_list_of_task_on_kart(var)}
+
+    def _is_list_of_task_on_kart(self, x):
```

Comment:
[IMO] 
I think it's better to include ` isinstance(var, TaskOnKart)` into `_is_list_of_task_on_kart ` .
(And of course rename the method)

Reply:
++

Reply:
Alright:)